TestFlightFailure_IgnitionFail.cs

```diff
@@ -148,6 +153,9 @@ public override float DoRepair()
                 EngineHandler engine = engines[i];
                 {
                     engine.engine.Shutdown();
```

Comment:
@jwvanderbeck, do you remember what the logic was behind making an ignitor repair shut down the engine? This was added in commit 1781eae8fc031eb78015afce8391adff7f4256aa.